sympy/functions/elementary/complexes.py

```diff
@@ -482,9 +482,9 @@ def eval(cls, arg):
                 return S.Infinity
         if arg.is_zero:
             return S.Zero
-        if arg.is_nonnegative:
+        if arg.is_nonnegative or arg.is_positive:
```

Comment:
If `arg.is_positive` is True then wouldn't `arg.is_nonnegative` be also True?

Reply:
Yes, but I guess I had put it just in case it wasn't taken care of in some other class.

Reply:
We should probably remove this. Unless you have an example/logic that justified having this.